lib/src/install.rs

```diff
@@ -165,11 +165,16 @@ pub(crate) struct InstallConfigOpts {
     #[serde(default)]
     pub(crate) generic_image: bool,
 
-    /// Do not pull any "logically bound" images at install time.
+    /// Do not resolve any "logically bound" images at install time.
     #[clap(long, hide = true)]
     #[serde(default)]
     pub(crate) skip_bound_images: bool,
```

Comment:
Maybe we should actually have
`--bound-images=stored|skip|pull` where `stored` is the existing default? (generated from an `enum` or so)

I don't recall exactly why `skip_bound_images` was added but it's `hide=true` so I think we can change it.

Reply:
Lol should've read this comment before refactoring the `Option` thing, yeah, let's do that

Reply:
Done, updated PR description and implementation